.stylelintignore

```diff
@@ -1,2 +1,3 @@
 packages/design-tokens/**/*.scss
 packages/components/locales/*
+storybook/styles/*
```

Comment:
A little iffy about ignoring all stylelint rules for the stylesheets just for probably one or two rules. Not too important though given it's local SB styles only, but might be worth revisiting later.